docs/multiclass_object_detector.md

```diff
@@ -0,0 +1,97 @@
+# Multi Class Object Detector
+
+Multi Class Object Detector is used when you want to detect and identify multiple classes of objects in one or more images.
+
+[Image to illustrate multi class object detector]
+
+In Xailient Console, you can train two popular multi-class object detectors:
+
+1. [SSD](https://arxiv.org/abs/1512.02325)
+2. [YOLO](https://arxiv.org/abs/1804.02767)
+
+To train a multi-class object detector with SSD architecture, you can either use the default training setting or define your own.
+
+The training configuration file gives you full control of the hyperparameters for the trainng.
+
+There are the setting you can modify:
+
+* __batch_size__: The batch size is a number of samples processed before the model is updated. The size of a batch must be more than or equal to one and less than or equal to the number of samples in the training dataset.
+
+* __epochs__: The number of epochs is the number of complete passes through the training dataset.  The number of epochs can be set to an integer value between one and infinity.
+
+* __augment_multiplier__: You can use this setting to augment your training data. 1 means add 1x to the dataset (double it). 2 means add 2x (triple it).
+
+We recommend keeping all the other setting to default.
+
+### SSD
+
+Here is a sample configuration for the SSD architeture. 
+
+``` yml
+data:
```

Comment:
I'm concerned about exposing config yml files to end users. The problem right now is that if there is a typo in the config.yml file, or some required value is missing, the training will simply fail and the user has no idea why. We (Xailient support) literally have to look at the training log files to figure out why the training failed; the user cannot figure this out on their own.
I've created https://app.asana.com/0/1162580331639025/1199992772008448/f to track this.